### ISSUE TITLE
IE10/IE11 setImmediate bug

### ISSUE THREAD

**Issue by dantman** [YuzuJS/setImmediate#35]
It looks like there is a bug inside IE10+'s implementation of setImmedate:
http://codeforhire.com/2013/09/21/setimmediate-and-messagechannel-broken-on-internet-explorer-10/

It might be desirable to ignore IE's setImmedate implementation and use the onreadystatechange trick for it.

**Comment by wswoodruff** [comment]
Ha! Found this in a random Google search `(Googled setimmediate ie10)`. Whaddup @dantman!

**Comment by ronkorving** [comment]
Any updates on this?

**Comment by domenic** [comment]
https://twitter.com/slicknet/status/782274190451671040

**Comment by ronkorving** [comment]
@dantman Were you planning on making a PR for this?

**Comment by dantman** [comment]
@ronkorving No, I don't believe I'm even using this specific polyfill implementation anymore.

**Comment by ronkorving** [comment]
@dantman Did you find a better alternative?

**Comment by dantman** [comment]
@ronkorving Lately I just use whole package polyfills like core-js. Though I haven't dug into their actual implementations or determined if they have this same issue.

**Comment by ronkorving** [comment]
@dantman Understood. Thanks 👍
